src/components/chart/_macro.njk

```diff
@@ -67,6 +67,8 @@
                     <figcaption class="ons-chart__caption">{{ params.caption }}</figcaption>
                 {% endif %}
             </figure>
+        {% else %}
+            <p><b>Chart type "{{ params.chartType }}" is not supported</b></p>
```

Comment:
We should probably add a test for this

Reply:
I've added one. I also changed it so that only the chart node was not rendered if the chart type is invalid - so we're showing all the other chart surroundings still (this is now more consistent as previously the title and subtitle were hidden, but the downloads still showed).

Reply:
Ive just noticed that I dont think we have tests for the title and id params, unless Ive missed them

Reply:
I'll check that now...

Reply:
The id check was there - I have added a check for the title. I've also remove the config for an invalid chart, and added more thorough testing for the invalid chart.

Reply:
Great! I can see the `data-highcharts-id` attribute being tested but not the `id` attribute

Reply:
I don't know why that `id` attribute was added as it isn't needed. I'd prefer to remove it rather than add a test for it. Is that ok as part of this PR or would it be better kept separate?

Reply:
Yeah just remove it in this PR then and mention it in the description

Reply:
Have you pushed? Doesn't look like the id has been removed. I was talking about the one added on line 25 in the macro

Reply:
I hadn't pushed - should be there now.